client/auth/Login.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { useRouter } from 'next/router';
+
+const Login = () => {
+  const router = useRouter();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleLogin = () => {
```

Comment:
Missing validation for email and password, within the handleLogin method. 

The function should handle the following case
1. missing email and password case
2. email format
3. password min length